packages/appstore/src/components/ready-to-update-wallets/upgrade-info-list.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Text, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
+import { TReadyToUpgradeForm } from './ready-to-upgrade-form';
+
+type TUpgradeInformationList = {
+    name: string;
+    visiblity: boolean;
+    content: React.ReactElement;
+};
+
+const UpgradeInformationList = ({ is_eu }: TReadyToUpgradeForm) => {
+    const list: TUpgradeInformationList[] = [
+        {
+            name: 'upgrade_info',
+            visiblity: true,
+            content: (
+                <Localize i18n_default_text='During the upgrade, deposits, withdrawals, transfers, and adding new accounts will be unavailable.' />
+            ),
+        },
+        {
+            name: 'open_positions',
+            visiblity: true,
+            content: (
+                <Localize i18n_default_text="Your open positions won't be affected and you can continue trading." />
+            ),
+        },
+        {
+            name: 'deriv_p2p',
+            visiblity: !is_eu,
```

Comment:
use the (`is_eu || is_eu_user`) instead of `is_eu`
```suggestion
            visiblity: !(is_eu || is_eu_user),
```
`is_eu_user`  can be extracted from traders-hub store

Reply:
yeah I know, the props is drilling down here from another compoenent which has access to store. I should just mentioned it here so he know where to search for it